lightbeam/lightbeam.py

```diff
@@ -174,7 +176,9 @@ def get_data_files_for_endpoint(self, endpoint):
         return file_list
 
     # Prunes the list of endpoints down to those for which .jsonl files exist in the config.data_dir
-    def get_endpoints_with_data(self):
+    def get_endpoints_with_data(self, filter_endpoints=None):
+        if not filter_endpoints:
+            filter_endpoints = self.all_endpoints
```

Comment:
This adds back the ability to filter down the `endpoints_with_data`.

Reply:
Might need to pull from main for these changes to be refelcted